packages/core/core/src/AssetGraphBuilder.js

```diff
@@ -158,6 +160,39 @@ export default class AssetGraphBuilder extends EventEmitter {
     assetGraph: AssetGraph,
     changedAssets: Map<string, Asset>,
   |}> {
+    let {config, configRef} = nullthrows(
+      await this.configRequestRunner.runRequest(null, {
```

Comment:
For now, the Parcel config file is being resolved twice (once in `Parcel.js` and once here). Once the `RequestGraph` supports sub-requests and is hoisted up the the `Parcel.js` level, then any request that requires the config will call that request directly, reusing the result from the graph as long as the config request is not invalid.